### PR TITLE
Loader fix in loader-master branch

### PR DESCRIPTION
**Pull request by iamtarun1993**

### CODE REVIEW

File: src/app/core/services/loader.service.ts [hunk 1]
```diff
@@ -7,17 +7,27 @@ import {noop} from 'rxjs';
 })
 export class LoaderService {
 
+  counter = 0;
+  loading;
+
   constructor(private loadingController: LoadingController) { }
 
   async showLoader(message = 'Please wait...', duration = 1000 ) {
```
Comment (iamtarun1993) [inline src/app/core/services/loader.service.ts:15]: for now `duration = 1000` is present in the method declaration, will remove this at last PR. Removing this now actually giving too much issue in ts compilation.

Though we defined this variable but never used it. check line no 20.

File: src/app/app.component.ts [hunk 2]
```diff
@@ -514,10 +514,10 @@ export class AppComponent implements OnInit {
 
     this.router.events.subscribe((ev) => {
       if (ev instanceof NavigationEnd) {
-        this.menuController.swipeGesture(false);
-        if ((ev.urlAfterRedirects.indexOf('enterprise') > -1) && !(ev.urlAfterRedirects.indexOf('delegated_accounts') > -1)) {
-          this.menuController.swipeGesture(true);
-        }
+        // this.menuController.swipeGesture(false);
```
Comment (Julias0) [inline src/app/app.component.ts:517]: ei why is this commented

Comment (Julias0) [reply src/app/app.component.ts:517]: dei why is this in PR

Comment (iamtarun1993) [reply src/app/app.component.ts:517]: will remove this when merging to master.

**Review by abhishek1234321 — CHANGES_REQUESTED** [review]

**Review by Julias0 — APPROVED** [review]

**Review by abhishek1234321 — APPROVED** [review]

**Pull request closed without merging**